### ISSUE TITLE
Allow global changes to all servers on AGDNS

### ISSUE THREAD

**Issue by jslawler** [AdguardTeam/AdGuardDNS#494]
It would be handy to have global changes that override all AGDNS servers.

Example, if I wanted Private Relay blocked on all servers, rather than having to tick the box for all servers, I could just change it once in the global settings.

**Comment by sofvaskina** [comment]
Hello! Thank you for your feature request. However, it is a duplicate of #641. Please check #641 for updates. You can also leave comments regarding the requested feature :)
I am closing your ticket as a duplicate.
